docker/postgres.override.yml

```diff
@@ -5,6 +5,7 @@ services:
   postgres:
     image: library/postgres:16-alpine
     container_name: intelowl_postgres
+    hostname: postgres
```

Comment:
why you need to rename them all?

Reply:
We had some with the name, some without. Just to have the same syntax everywhere